controllers/vrg_volrep_test.go

```diff
@@ -1360,6 +1421,44 @@ func (v *vrgTest) promoteVolReps() {
 	}
 }
 
+func (v *vrgTest) deleteProtectVolReps() {
```

Comment:
If you could  rephrase  deleteProtcectVolReps to ProtectVolRepsFromDeletion or something similar it would be better.

Reply:
I chose the current scheme based on `promoteVolReps` function, hence the naming. If particular I can change it to possibly:`protectDeletionOfVolReps` and `unprotectDeletionOfVolReps` (the counterpart)

Reply:
Done.